File: ChallengeBackend/src/main/java/br/com/savedra/challengebackend/model/Campaign.java
package br.com.savedra.challengebackend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "campaigns")
public class Campaign {
    @Id
    private String id;
    private String title;
    private String description;
    private String segment;
    private String startDate;
    private String endDate;
}

File: ChallengeCRM/app/src/main/java/br/com/savedra/challengecrm/model/Campaign.kt
package br.com.savedra.challengecrm.model

/**
 * Modelo de Campanha para o Android.
 */
data class Campaign(
  val id: String? = null,
  val title: String? = "",
  val description: String? = "",
  val startDate: String? = "",
  val endDate: String? = "",
  val segment: String? = ""
)
